docs/DESIGN-SYSTEM.md

```
@@ -391,22 +453,25 @@ section {
 }
 
 .primary:hover {
-  background-color: var(--color-primary-dark); /* #6D28D9 */
-  transform: translateY(-2px);
+  background-color: var(--color-primary-dark); /* #000000 (light) or #f1f5f9 (dark) */
+  transform: translateY(-1px);
   box-shadow: var(--shadow-md);
 }
 ```
 
 **2. Secondary Button**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)